pkg/dataflatten/flatten.go

```diff
@@ -227,6 +227,13 @@ func (fl *Flattener) descend(path []string, data interface{}, depth int) error {
 				return errors.Wrap(err, "flattening map")
 			}
 		}
+	case []map[string]interface{}:
```

Comment:
Yea, I thought that `[]map[string]interface{}` would be considered an `[]interface{}` ... I spend a little time seeing if we could leverage generics to make this cleaner. I don't like using `interface{}` so much. Right now you [can't type switch on a generic parameter](https://go.dev/play/p/KAB1hWiV9zI?v=gotip.go?download=true.go?download=true) so I don't know how useful it would be. However, there is [discussion](https://github.com/golang/go/issues/49206) about it being added in a later versions.